src/Components/Patient/PatientInfoCard.tsx

```diff
@@ -461,6 +466,22 @@ export default function PatientInfoCard(props: {
                       icon="l-check"
                       className="ml-2 fill-current text-xl text-green-500"
                     />
+                    {skills?.results?.slice(0, 2).map((item, index) => (
+                      <div key={index} className="">
+                        {index === 1 ? (
+                          <div className="mr-2">{item.skill_object.name}</div>
+                        ) : (
+                          <div className="mr-2">{item.skill_object.name},</div>
+                        )}
+                      </div>
+                    ))}
+                    {skills?.results?.slice(2)?.length ?? 0 > 0 ? (
+                      <div>
+                        and {skills?.results?.length ?? 0 - 2} other skills
+                      </div>
+                    ) : (
+                      <div>.</div>
+                    )}
```

Comment:
This is not the right way to show continuous text. Use `span` instead of `div` tags. Could you simplify the logic too? The code isn't readable.

Reply:
Sure... I will change to span tags from the divs... 

So my logic behind the code goes like:
First it takes 2 skills and prints out separated by a comma
and then it checks that if there are skills left after the two skills , (nullish coalescing), it prints the number followed by the text "other skills" and if there aren't skills apart from the two shown, it simply prints full-stop "."

Reply:
Would be cleaner if you had a function something like this.

```js
const getSkillsDescription: (skills: SkillModel[]) => {
  if (skills.length === 1) {
    return skills[0].skill_object.name;
  }

  if (skills.length === 2) {
    return skills.map(s => s.skill_object.name).join(' and ');
  }

  return skills.slice(0, 2).map(s => s.skill_object.name).join(', ') + `${skills.length - 2} more`;
}
```

Reply:
have a look at the updated PR